barretenberg/cpp/src/barretenberg/vm/avm/trace/trace.cpp

```diff
@@ -3472,7 +3472,7 @@ std::vector<Row> AvmTraceBuilder::finalize(uint32_t min_trace_size, bool range_c
     // Main Trace needs to be at least as big as the biggest subtrace.
     // If the bin_trace_size has entries, we need the main_trace to be as big as our byte lookup table (3 *
     // 2**16 long)
-    size_t const lookup_table_size = (bin_trace_size > 0 && range_check_required) ? 3 * (1 << 16) : 0;
+    size_t const lookup_table_size = range_check_required ? 3 * (1 << 16) : 0;
```

Comment:
Not for this PR, but it seems `range_check_required` isnt quite accurate anymore after this change.

IIUC we now either have "minimal" lookup tables (of just read values) or the complete tables. Perhaps we need to update this flag to reflect this (maybe `testing_config`)